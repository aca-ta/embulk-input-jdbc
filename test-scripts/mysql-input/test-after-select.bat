mysql -uTEST_USER -pXXXXXXXX -DTESTDB < test.sql

CALL embulk run test-after-select.yml

echo "diff data/test_expected.csv data/test000.00.csv"
diff data/test_expected.csv data/test000.00.csv

IF "%ERRORLEVEL%" == "0" (ECHO "OK!") ELSE (ECHO "embulk-input-mysql (after-select) FAILED!")

mysql -uTEST_USER -pXXXXXXXX -DTESTDB -e"SELECT * FROM EMBULK_INPUT" > data/temp.txt

echo "diff data/test-after-select_expected.txt data/temp.txt"
diff data/test-after-select_expected.txt data/temp.txt

IF "%ERRORLEVEL%" == "0" (ECHO "OK!") ELSE (ECHO "embulk-input-mysql (after-select) FAILED!")
