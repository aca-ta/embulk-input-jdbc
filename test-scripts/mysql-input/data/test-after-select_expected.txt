ID	NUM	STR	VARSTR	DT	DTTM0	DTTM3
1	123.40	xx	TEST1	2015-04-24	2015-04-24 01:02:03	2015-04-24 01:02:03.123
2	1234567890.12	xx	TEST9999	2015-12-31	2015-12-31 23:59:59	2015-12-31 23:59:59.999
3	NULL	xx	NULL	NULL	NULL	NULL
